cmd/juju/controller/listmodels.go

```diff
@@ -100,7 +98,6 @@ func (c *modelsCommand) Run(ctx *cmd.Context) error {
 	}
 	if !names.IsValidUser(c.user) {
 		err := errors.NotValidf("user %q", c.user)
-		ctx.Infof(err.Error())
 		return err
```

Comment:
err was created for Infof. you can take this a step further and reduce to `return errors.NotValidf("user %q", c.user)`

Reply:
sure! thanks for the catch!